internal/config/main_config.go

```diff
@@ -74,6 +77,7 @@ var AWS = &config.AWS
 var Features = &config.FeatureFlags
 var ImageBuilderURL = &config.RestEndpoints.ImageBuilder.URL
 var SourcesURL = &config.RestEndpoints.Sources.URL
+var OpenAPIPath = &config.OpenAPI.Path
```

Comment:
Yeah I agree that exporting the final config options might lead to a lot of variables for no reason.
With this suggestion you write `OpenAPI.Path` instead of `OpenAPIPath`, which is maybe even bit more readable :)

Reply:
I agree, we will end with many vars.
I'll fix the other vars in a separate PR for easy testing :)

Reply:
Sure, I just wanted to comment on this one :)

Reply:
we are addressing the rest in #60